test/plugin/test_parser_syslog.rb

```diff
@@ -294,14 +294,14 @@ def test_parse_with_rfc5424_structured_message
                         'message_format' => 'rfc5424',
                         'with_priority' => true,
                         )
-      text = '<16>1 2017-02-06T13:14:15.003Z 192.168.0.1 fluentd 11111 ID24224 [exampleSDID@20224 iut="3" eventSource="Application" eventID="11211"] Hi, from Fluentd!'
+      text = '<16>1 2017-02-06T13:14:15.003Z 192.168.0.1 fluentd 11111 ID24224 [exampleSDID@20224 iut="3" eventSource="Application" eventID="11211"] [Hi] from Fluentd!'
```

Comment:
I looked over the RFC5424 ABFN.
it shows like below. This means that `[hi]` should be the `SD-ELEMENT`,  not `MSG`,  because `1*SD-ELEMENT` means at least one `SD-ELEMENT` is allowed.

```
 STRUCTURED-DATA = NILVALUE / 1*SD-ELEMENT
 SD-ELEMENT      = "[" SD-ID *(SP SD-PARAM) "]"
```